integrations/build_system/xmake.rst

```diff
@@ -0,0 +1,69 @@
+.. xmake:
+
+XMake
+=========
+
+
+Install third-party packages:
+-------
+
+After version 2.2.5, xmake supports support for dependency libraries in third-party package managers.
+
+.. code-block:: lua
+   :caption: xmake.lua
+    
+    -- xmake.lua
+    
+    add_requires("CONAN::zlib/1.2.11@conan/stable", {alias = "zlib", debug = true})
+    add_requires("CONAN::OpenSSL/1.0.2n@conan/stable", {alias = "openssl",
+        configs = {options = "OpenSSL:shared=True"}})
+    
+    target("test")
+        set_kind("binary")
+        add_files("src/*.c") 
+        add_packages("openssl", "zlib")
+
+
+After executing xmake to compile:
+
+.. code-block:: bash
+
+    $ xmake
+    checking for the architecture ... x86_64
+    checking for the Xcode directory ... /Applications/Xcode.app
+    checking for the SDK version of Xcode ... 10.14
+    note: try installing these packages (pass -y to skip confirm)?
+      -> CONAN::zlib/1.2.11@conan/stable  (debug)
+      -> CONAN::OpenSSL/1.0.2n@conan/stable  
+    please input: y (y/n)
+
+      => installing CONAN::zlib/1.2.11@conan/stable .. ok
+      => installing CONAN::OpenSSL/1.0.2n@conan/stable .. ok
+
+    [  0%]: ccache compiling.release src/main.c
+    [100%]: linking.release test
+
+
+Find the conan package
+-------
+
+XMake v2.2.6 and later versions also support finding the specified package from the conan:
```

Comment:
from the conan =>  finding the specified package in the Conan cache (does it find it in the cache or in remote repositories?)